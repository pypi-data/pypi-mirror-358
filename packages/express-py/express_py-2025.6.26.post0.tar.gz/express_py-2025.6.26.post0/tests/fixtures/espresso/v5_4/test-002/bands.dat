 &plot nbnd=   8, nks=     6 /
            0.000000  0.000000  0.000000
  -5.597   6.272   6.272   6.272   8.713   8.713   8.713   9.419
            0.288675  0.204124 -0.500000
  -3.300  -0.663   5.063   5.063   7.698   9.494   9.494  13.901
            0.000000 -0.612372  0.000000
  -3.300  -0.663   5.063   5.063   7.698   9.494   9.494  13.901
            0.288675 -0.408248 -0.500000
  -1.508  -1.508   3.413   3.413   6.922   6.922  16.150  16.150
           -0.577350  0.204124  0.000000
  -3.300  -0.663   5.063   5.063   7.698   9.494   9.494  13.901
           -0.577350 -0.408248  0.000000
  -1.508  -1.508   3.413   3.413   6.922   6.922  16.150  16.150
