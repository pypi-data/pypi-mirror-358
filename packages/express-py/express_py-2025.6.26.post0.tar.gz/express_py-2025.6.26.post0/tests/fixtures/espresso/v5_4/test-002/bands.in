&BANDS
    prefix = '__prefix__'
    outdir = '/home/exadmin/data/exadmin-default/new-job-aug-21st-2017-18-47-pm-cmF9STjvfyTkoJcte/outdir'
    filband = '/home/exadmin/data/exadmin-default/new-job-aug-21st-2017-18-47-pm-cmF9STjvfyTkoJcte/bands.dat'
    no_overlap = .true.
/
