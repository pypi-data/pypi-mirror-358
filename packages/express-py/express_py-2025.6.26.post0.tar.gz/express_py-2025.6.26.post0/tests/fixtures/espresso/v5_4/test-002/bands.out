
     Program BANDS v.5.4.0 starts on 22Aug2017 at  1:48:10

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
          URL http://www.quantum-espresso.org",
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Serial version

     Reading data from directory:
     /home/exadmin/data/exadmin-default/new-job-aug-21st-2017-18-47-pm-cmF9STjvfyTkoJcte/outdir/__prefix__.save

   Info: using nr1, nr2, nr3 values from input

   Info: using nr1, nr2, nr3 values from input

     IMPORTANT: XC functional enforced from input :
     Exchange-correlation      = SLA  PW   PBE  PBE ( 1  4  3  4 0 0)
     Any further DFT definition will be discarded
     Please, verify this is what you really want


     G-vector sticks info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Sum         745     583    169                13323     9409    1471

     high-symmetry point:  0.0000 0.0000 0.0000   x coordinate   0.0000
     high-symmetry point:  0.2887 0.2041-0.5000   x coordinate   0.6124
     high-symmetry point:  0.0000-0.6124 0.0000   x coordinate   1.6124
     high-symmetry point:  0.2887-0.4082-0.5000   x coordinate   2.2247
     high-symmetry point: -0.5774 0.2041 0.0000   x coordinate   3.3973
     high-symmetry point: -0.5774-0.4082 0.0000   x coordinate   4.0097

     Plottable bands written to file /home/exadmin/data/exadmin-default/new-job-aug-21st-2017-18-47-pm-cmF9STjvfyTkoJcte/bands.dat.gnu
     Bands written to file /home/exadmin/data/exadmin-default/new-job-aug-21st-2017-18-47-pm-cmF9STjvfyTkoJcte/bands.dat

     BANDS        :     0.64s CPU         0.70s WALL


   This run was terminated on:   1:48:11  22Aug2017

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
